deep_quoridor/src/arena.py

```diff
@@ -125,20 +125,24 @@ def _play_game(self, agent1: Agent, agent2: Agent, game_id: str) -> GameResult:
         self.game.close()
         return result
 
-    def play_games(self, players: list[str], times: int):
+    def play_games(self, players: list[str | Agent], times: int):
         self.plugins.start_arena(self.game)
 
         match_id = 1
         results = []
+        agents = []
+        for p in players:
+            if isinstance(p, Agent):
+                agents.append(p)
+            else:
+                agents.append(AgentRegistry.create(p))
```

Comment:
@adamantivm , I added this so that we can pass an instance instead of the friendly name, that way we can instantiate it however we want.  It still doesn't allow to create custom instances from the command line, but it may be useful